File: Codeforces-Sheet-level(0)-mansoura/sheet2/problem-B/solution.java
import java.util.Scanner;
public class solution{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n,x;
        n = sc.nextInt();
        x = sc.nextInt();

        int num; 
        for(int i=0 ; i<n ; i++){
            num = sc.nextInt();
            if(num == x){
                System.out.println(i);
                return;
            }
        }

        System.out.println("Not Found");
    }
}

File: Codeforces-Sheet-level(0)-mansoura/sheet2/problem-B/solution.cpp
#include <iostream>
using namespace std;

int main(){
    int n,x;
    cin>>n>>x;

    int num;
    for(int i=0; i<n ;i++)
    {
        cin>> num;
        if(num==x)
        {
            cout<< i;
            return 0;
        }
    }

    cout<< "Not Found";

    return 0;
}